### ISSUE TITLE
Bug: wrong manual backport message (not always)

### ISSUE THREAD

**Issue by wolfogre** [GiteaBot/gitea-backporter#94]
https://github.com/go-gitea/gitea/pull/25721#issuecomment-1624581483

**Comment by yardenshoham** [comment]
Yes, the bot uses labels for state, and seeing as we only have one `backport/done` label the bot doesn't add it if a PR has more than one `backport/v1.*` labels. I want to keep the bot stateless, so this is the situation. If we had `backport/v1.*-done` labels we could maintain state in the labels.

**Comment by KN4CK3R** [comment]
Can't you check the reason if cherry picking fails to check if all backports are done?

**Comment by yardenshoham** [comment]
Currently we simply log the `stdout` and `stderr` of the `git cherry-pick`

**Comment by KN4CK3R** [comment]
Yes, but doesn't the output give a clue why it failed which can be used by the bot?

**Comment by yardenshoham** [comment]
Possibly, usually it lists the conflicted files
